apps/dashboard/src/components/primitives/multi-select.tsx

```diff
@@ -17,6 +17,7 @@ export const MultiSelect = <T extends string | number>({
   placeholderAll,
   className,
   onValuesChange,
+  size = 'default',
```

Comment:
this component I created some time ago, but it wasn't used anywhere, so I've updated it to match the designs and it's used in the Vercel integration form